### DISCUSSION TITLE
Custom feature docs clarification

### DISCUSSION THREAD

**Discussion by ambid17** [TanStack/table#5575] [Q&A]
I am trying to create a custom feature that allows the user to bulk edit multiple cells in the same column. The goal is to bring in the copy/paste functionality from Excel: where you can select multiple cells, copy them, and then paste the contents of that block of cells in another area of the table.

I was following the "Custom Features Guide" but it appears Typescript is unhappy about extending "TableOptionsResolved".

![image](https://github.com/TanStack/table/assets/12086352/ea9784d9-93c5-49e5-9e8d-dd185628a916)

So, two questions:
- does anyone have an example laying around for what I am trying to do that would work with V8? 
I am basically trying to migrate this guy to V8: https://github.com/TanStack/table/discussions/2476
- is this a problem with my Typescript configuration, or is this an issue with the documentation?

**Comment by KevinVandy** [comment]
The generics must match in order for declaration merging to work. Make sure it is: 
`interface TableOptionsResolved<TData extends RowData>`
and not
`interface TableOptionsResolved`
